cmd/controller-manager/app/controller_manager.go

```diff
@@ -34,9 +35,13 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/client-go/tools/record"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/apiserver/pkg/server/healthz"
+	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
```

Comment:
Should probably prefer to use pkg `k8s.io/client-go/kubernetes` instead of this. This is because, over time, we should be looking to reduce dependencies on the monolithic `k8s.io/kubernetes` and prefer the smaller libraries that don't drag as many transitive dependencies along with them. (Ideally, we'd consume leader election machinery from somewhere other than `k8s.io/kubernetes` as well, but that doesn't exist elsewhere yet. For now, take what we can get.)